### PR TITLE
Update/accordion

### PR DESCRIPTION
**Pull request by darintrimillos**
This includes the breaking changes release notes as well as normal docs updates for accordion refactor.

Do the breaking changes notes seem clear enough?

### CODE REVIEW

File: docs/release-notes/fall-2019/README.md [hunk 1]
```diff
@@ -22,6 +22,10 @@
 
 ## Breaking Changes
 
+### Stateless Accordion
+
+CdrAccordion has been refactored to remove internal state tracking for toggling open and closed. For simplicity, the CdrAccordionItem component has been removed and its functionality merged with CdrAccordion. The `show` prop has been renamed to `opened` for clarity. Please review the [updated API and usage examples](https://rei.github.io/rei-cedar-docs/components/accordion/) for more information.
+
```
Comment (cowills) [inline docs/release-notes/fall-2019/README.md]: can you add a simple before and after example here?

**Comment by cowills** [comment]
Can you also update NavLinks.vue to use the new accordion API? will probably need to cut a pre-release alpha to do that.

File: package.json [hunk 2]
```diff
@@ -31,7 +31,7 @@
     "vuepress": "^1.0.2"
   },
   "dependencies": {
-    "@rei/cedar": "^3.0.0-alpha.2",
+    "@rei/cedar": "^3.0.0-accordion.alpha.1",
```
Comment (cowills) [inline package.json]: 3.0.0-alpha.2 already had the accordion updates in it. Does your alpha have my v-model changes in it?

Comment (darintrimillos) [reply package.json]: Yeah, sorry for the confusion here. I found a bug in the accordion code while trying to update the nav with `alpha.2`. That PR can be found here: https://github.com/rei/rei-cedar/pull/662

Comment (cowills) [inline package.json]: looks like the cedar version got reverted when you merged from next

**Comment by cowills** [comment]
@darintrimillos  whats the status of this PR? I think we'll need at least the release notes part of this PR merged before we release the beta to consumers

**Pull request closed without merging**